ingestion-beam/src/test/java/com/mozilla/telemetry/util/NormalizeTest.java

```diff
@@ -44,4 +44,64 @@ public void channel() {
     assertEquals("beta", Normalize.channel("beta-cdntest"));
     assertEquals("beta", Normalize.channel("betastop"));
   }
+
+  @Test
+  public void os() {
+    // Desktop OS.
+    assertEquals("Windows", Normalize.os("Windows"));
+    assertEquals("Windows", Normalize.os("WINNT"));
+    assertEquals("Windows", Normalize.os("Windows_NT"));
+    assertEquals("Windows", Normalize.os("WindowsNT"));
+    assertEquals("Mac", Normalize.os("Darwin"));
+    assertEquals("Linux", Normalize.os("Linux"));
+    assertEquals("Linux", Normalize.os("GNU/Linux"));
+    assertEquals("Linux", Normalize.os("SunOS"));
+    assertEquals("Linux", Normalize.os("FreeBSD"));
+    assertEquals("Linux", Normalize.os("GNU/kFreeBSD"));
+    assertEquals("Other", Normalize.os("AIX"));
+
+    // Mobile OS.
+
```

Comment:
this seems like it needs either at least one test or removal

Reply:
Totally spaced on that. Will fix.